### DISCUSSION TITLE
On desktop mj-column is stacked in thunderbird and other webmail based clients

### DISCUSSION THREAD

**Discussion by webthingz** [mjmlio/mjml#2912] [Q&A]
Hi All,

I have a template with a 3 column section. This working fine on mobile, roundcube, gmail and outlook. But it is not working on thunderbird en my own webmail client from Axigen. I tried with mj-group and then the columns are good on de desktop but the columns need to be stacked on mobile. Breaking my head on this.. 

Greetings! Ziggy

**Comment by iRyusa** [comment]
Try to send with https://putsmail.com/ it shouldn't happen if the HTML isn't altered
